services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py

```diff
@@ -100,7 +117,7 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
     )
 
     DYNAMIC_SIDECAR_RESERVED_SPACE_SIZE: ByteSize = Field(
-        parse_obj_as(ByteSize, "10Mib"),
+        TypeAdapter(ByteSize).validate_python("10Mib"),
```

Comment:
You created a ByteSizeAdapter, why don't you use it?

Reply:
I've replaced it globally